test/atmos/atmosphere.jl

```diff
@@ -0,0 +1,42 @@
+using Terrarium
+using Test
+
+using Oceananigans: Field
+
+@testset "AtmosphericState" begin
+    grid = ColumnGrid(ExponentialSpacing())
+    # use nonzero test values to check that everything gets updated correctly
+    precip = TwoPhasePrecipitation(rainfall=1.0, snowfall=1.0)
+    solar = TwoBandSolarRadiation(shortwave=1.0, longwave=1.0)
+    T_air = 1.0
+    humidity = 1.0
+    pressure = 1.0
+    windspeed = 1.0
+    tracers = TracerGases(AmbientCO2(1.0))
+    atm = AtmosphericState(; grid, precip, solar, T_air, humidity, pressure, windspeed, tracers)
+    @test haskey(atm.tracers, :CO2)
+    atm_varnames = map(Terrarium.varname, variables(atm))
+    @test all(map(∈(atm_varnames), (:T_air, :humidity, :pressure, :windspeed, :rainfall, :snowfall, :SwIn, :LwIn)))
+    # check compute_auxiliary!
+    state = (
+        T_air = Field{Center,Center,Nothing}(get_field_grid(grid)),
```

Comment:
We'd always use a Oceanigans.Field here? 

Then, we would interpolate between the RingGrids.Field and the Oceanigans.Field before?